<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plot_gui</class>
 <widget class="QMainWindow" name="plot_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Визуализатор графиков</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="5">
     <widget class="QDoubleSpinBox" name="Xmax">
      <property name="minimum">
       <double>-999999.000000000000000</double>
      </property>
      <property name="maximum">
       <double>999999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QDoubleSpinBox" name="Ymin">
      <property name="minimum">
       <double>-999999.000000000000000</double>
      </property>
      <property name="maximum">
       <double>999999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QDoubleSpinBox" name="Ymax">
      <property name="minimum">
       <double>-999999.000000000000000</double>
      </property>
      <property name="maximum">
       <double>99999999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="9">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <widget class="QDoubleSpinBox" name="Xmin">
      <property name="minimum">
       <double>-999999.000000000000000</double>
      </property>
      <property name="maximum">
       <double>999999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Xmax</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Ymax</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" rowspan="3" colspan="11">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QCustomPlot" name="customPlot" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="10">
     <widget class="QCheckBox" name="cbLegend">
      <property name="text">
       <string>Легенда</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Ymin</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Xmin</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_openfile"/>
    <addaction name="separator"/>
    <addaction name="action_clear"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <addaction name="action_function"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="action_openfile">
   <property name="text">
    <string>Открыть файл...</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_clear">
   <property name="text">
    <string>Очистить</string>
   </property>
  </action>
  <action name="action_function">
   <property name="text">
    <string>Ввод из строки..</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
